module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -578,16 +690,16 @@ function ($limiter) {
                 );
                 return $limiter;
             },
-            $this->limiterOptions
+            $this->getApiProperty('limiterOptions')
         );
         // No setting = use all available values
-        if (empty($this->advancedLimiters)) {
+        if (!($advancedLimiters = $this->getApiProperty('advancedLimiters'))) {
```

Comment:
Do we need to use getApiProperty here? This appears to only be set by configuration. (Though if you think it's safer to just use getApiProperty everywhere to be safe, I wouldn't object too strongly...)